frontend/src/components/pages/ProfileCompletionTwo.jsx

```diff
@@ -0,0 +1,257 @@
+import LeftBar from "./ProfileCompletion/LeftBarProfile";
+import SkipButton from "./ProfileCompletion/SkipButton";
+import NextButton from "./ProfileCompletion/nextButton";
+import ProgressBar from "./ProfileCompletion/progressBar";
+import UrapolkuLogo from "./ProfileCompletion/UrapolkuLogoText";
+import "../css/ProfileCompletionTwo.css";
+import { useState } from "react";
+function ProfileCompletionTwo() {
+  const [openIndustry, setOpenIndustry] = useState(false);
+  const [openSkills, setOpenSkills] = useState(false);
+  const [clickedSkills, setClickedSkills] = useState([]);
+
+  const handleIndustryClick = (event) => {
+    // This gets the text-content when industry children is clicked
+    // It stores the value into IndustryDataValue
+    const IndustryDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".Industry-select-text");
```

Comment:
reactissa pitäis teoriassa käyttää useRef hookkia näiden selector hakujen tilalla